docs/make.jl

```diff
@@ -113,6 +113,7 @@ makedocs(;
                 "Generalized Stiefel" => "manifolds/generalizedstiefel.md",
                 "Generalized Grassmann" => "manifolds/generalizedgrassmann.md",
                 "Grassmann" => "manifolds/grassmann.md",
+                "Hamiltonian" => "manifolds/hamiltonian.md",
```

Comment:
Is it a separate manifold?

Reply:
I don't see a separate `<:AbstractManifold` there.

Reply:
Not yet. See list above. It just makes sense for consistency to introduce it, it will be super similar to symmetric (a Euclidean sub manifold with the Euclidean metric).

Reply:
Unless you are against that, but my two comparison manifolds are symmetric and skew symmetric – not super spectacular manifolds, but sometimes nice to have. The same holds here.

Reply:
This is completely fine.

Reply:
Ok, I will add that then, again, not super spectacular, but maybe useful for someone.